tests/jvm/src/test/resources/metac.expect

```diff
@@ -288,6 +288,141 @@ Synthetics:
   [0:0..0:1): * => _star_.
   [0:2..0:6): Unit => scala.Unit#
 
+semanticdb/integration/src/main/scala/example/Annotations.scala
+---------------------------------------------------------------
+
+Summary:
+Schema => SemanticDB v4
+Uri => semanticdb/integration/src/main/scala/example/Annotations.scala
+Text => non-empty
+Language => Scala
+Symbols => 19 entries
+Occurrences => 61 entries
+Synthetics => 1 entries
+
+Symbols:
+annot. => package annot
+annot.Annotations# => @SerialVersionUID class Annotations[@specialized T] extends AnyRef { self: AnyRef => +5 decls }
+SerialVersionUID => scala.SerialVersionUID#
+T => annot.Annotations#[T]
+specialized => scala.specialized#
+AnyRef => scala.AnyRef#
+annot.Annotations#T# => @SerialVersionUID abstract type T
+SerialVersionUID => scala.SerialVersionUID#
+annot.Annotations#[T] => @specialized typeparam T
+specialized => scala.specialized#
+annot.Annotations#`<init>`(). => primary ctor <init>(@deprecatedName x: T)
+x => annot.Annotations#`<init>`().(x)
+deprecatedName => scala.deprecatedName#
+T => annot.Annotations#[T]
+annot.Annotations#`<init>`().(x) => @deprecatedName param x: T
+deprecatedName => scala.deprecatedName#
+T => annot.Annotations#[T]
+annot.Annotations#field(). => val method field: Int
```

Comment:
Where did the annotation go? This should say `@throws[Exception] va method field: Int` if I understand correctly.

Reply:
Could the annotation be attached to the private[this] field symbol instead of the getter symbol?